### ISSUE TITLE
should be able to see order summary at any time

### ISSUE THREAD

**Issue by ahdinosaur** [root-systems/cobuy#389]
especially admins are keen to see what the order summary would look like if sent to suppliers

**Comment by michael-smith-nz** [comment]
Looks like the only change will need to use `orderIntents` instead of `orderPlans` as the data source. My understanding is that they are the same data shape

**Comment by iainkirkpatrick** [comment]
butting in here for a quick thought - i'm not sure my head is fully on this / understanding the purpose, but AFAIK it won't just be a case of switching out `orderPlans` for `orderIntents` - there is always only one `OrderPlan` created for each unique combination of an agent, product and order, whereas there may be many `Intents` from an agent per product per order. I.e. I can cast intents at all the price points on bananas in an order, but only one of those is the 'true' price point at any given time (depending on the intents of the rest of the group).

There is logic (read. getters) to figure out the 'true' price point on the client, used in the product list view to show the correct price and quantity... my bet would be to use those getters in this kind of pre-emptive OrderSummary view

sorry if this has already been covered and i'm just behind :sweat_smile:

**Comment by iainkirkpatrick** [comment]
@NotThatSmith you are off this ticket now right? can i take it over?

**Comment by danalexilewis** [comment]
@iainkirkparick yes Michael has shifted all hit work to the “tech between us” work for now. So take it over :)

**Comment by iainkirkpatrick** [comment]
@agentlewis in light of sprint planning this morning, is this still relevant?

**Comment by danalexilewis** [comment]
Yes but second milestone work.
